feat(react-scheduler): add capability to set up auto-scroll speed (#2939)
DevExpress/devextreme-reactive: packages/dx-react-scheduler/docs/reference/drag-drop-provider.md

-----|------|---------|------------
allowDrag? | (appointmentData: [AppointmentModel](./scheduler.md#appointmentmodel)) => boolean | () => true | A function that specifies draggable appointments.
allowResize? | (appointmentData: [AppointmentModel](./scheduler.md#appointmentmodel)) => boolean | () => true | A function that specifies resizable appointments.

> scrollSpeed? | number | 15 | Specifies the scroll speed when resizing or dragging an appointment.

Reviewer: > Specifies the scroll speed when resizing or dragging an appointment.

Specifies the auto-scroll speed while appointment drag-and-drop manipulations.

Reviewer 2: ```suggestion
scrollSpeed? | number | 15 | Specifies the scroll speed during appointment resize or drag and drop.
```
